meta/system.py

```diff
@@ -166,6 +166,15 @@ def set_kaldi_scorer(self, corpus, mapping):
         }
         self.scorer_hyp_arg[corpus] = "ctm"
 
+    def set_stm_files(self, corpus, **kwargs):
+        """
+        create and set the stm files
+        :param Path|str corpus: Bliss corpus file path
```

Comment:
But this is wrong now or not?
This should be just the corpus "name" as string, right?

Reply:
ah yes you are right..